tests/pebble-fetch.sh

```diff
@@ -13,15 +13,31 @@ mkdir -p ${PEBBLEPATH}
 
 cat << UNLIKELY_EOF > "$PEBBLEPATH/docker-compose.yml"
 version: '3'
-
 services:
- pebble:
-  image: letsencrypt/pebble:${PEBBLE_VERSION}
-  command: pebble -strict ${PEBBLE_STRICT:-false} -dnsserver 10.77.77.1
```

Comment:
Did the behavior around `-strict` change or did this environment variable never work? If we're not going to allow people to set this variable to enable strict mode, can we remove `PEBBLE_STRICT` from `tox.ini`?

Reply:
Tomorrow, pebble will use the strict mode by default, and I saw that in some situation, pebble in non strict mode was more permissive than Boulder, that is not useful.

So yes, this environnement variable is not useful anymore, and we can remove `PEBBLE_STRICT` from `tox.ini`, without consequences because `apacheconftest` does not issue certificates using pebble.

Reply:
Done